config/config.go

```diff
@@ -253,23 +253,32 @@ const (
 )
 
 type InfluxMetrics struct {
-	Host     string `mapstructure:"host"`
-	Database string `mapstructure:"database"`
-	Username string `mapstructure:"username"`
-	Password string `mapstructure:"password"`
-	Enabled  bool   `mapstructure:"enabled"`
+	Enabled         bool   `mapstructure:"enabled"`
+	Host            string `mapstructure:"host"`
+	Database        string `mapstructure:"database"`
+	Measurement     string `mapstructure:"measurement"`
+	Username        string `mapstructure:"username"`
+	Password        string `mapstructure:"password"`
+	AlignTimestamps bool   `mapstructure:"align_timestamps"`
```

Comment:
Should these two new values have to be added to `func setConfigDefaults(v *viper.Viper) `?